packages/flutter_tts/tizen/src/text_to_speech.cc

```diff
@@ -109,13 +121,24 @@ std::vector<std::string> &TextToSpeech::GetSupportedLanaguages() {
   return supported_lanaguages_;
 }
 
-tts_state_e TextToSpeech::GetState() {
+TtsState TextToSpeech::GetState() {
   tts_state_e state;
   int ret = tts_get_state(tts_, &state);
   if (ret != TTS_ERROR_NONE) {
-    LOG_ERROR("[TTS] tts_prepare failed: %s", get_error_message(ret));
+    LOG_ERROR("[TTS] tts_get_state failed: %s", get_error_message(ret));
+    return TtsState::kError;
+  }
+  switch (state) {
+    case TTS_STATE_CREATED:
+      return TtsState::kCreated;
+    case TTS_STATE_READY:
+    default:
```

Comment:
Just one suggestion based on style guide: https://google.github.io/styleguide/cppguide.html#Loops_and_Switch_Statements.
> If the default case should never execute, treat this as an error.

Reply:
Thank you for review. I updated the commit.